src/main/java/academicweapon/task/Event.java

```diff
@@ -0,0 +1,48 @@
+package academicweapon.task;
+
+/**
+ * Represents an event task in the Duke application.
+ * The Event class extends the Task class and includes a start and end time for the event.
+ */
+public class Event extends Task {
+
+    protected String from;
```

Comment:
Maybe you can be more specific such as fromTime and i think it needs to be stored as localdate or localdatetime format instead, same as your deadline class
![image](https://github.com/nus-cs2103-AY2324S2/ip/assets/95204474/576f60b8-3db3-45e4-bc81-f5b94dd2fd11)
It would also be better to be able to implement certain localdate features such as sorting based on time